### ISSUE TITLE
Raw mode is not supported on the current process.stdin

### ISSUE THREAD

**Issue by konsumer** [cloudflare/wrangler2#718]
#### Describe the bug

I have a script that migrates env-vars on deploy, like this:

```
echo $MY_SECRET | npx -y wrangler@alpha --env main secret put MY_SECRET
```
I get this error, now:
```
 ERROR  Raw mode is not supported on the current process.stdin, which Ink uses as input stream by default.
        Read about how to prevent this error on https://github.com/vadimdemedes/ink/#israwmodesupported
```

#### Expected behavior

There should be a way to set secrets, in scripts (without user-input.)

#### Environment and versions

Fill out the following information about your environment.

* operating system: OSX 12.2 (Monterey)
* output of `wrangler -v`: 0.0.0-046b17d
* output of `node -v`: v17.6.0

**Comment by threepointone** [comment]
This was fixed in 0.0.20 with #559. Could you try the latest version and see if it works for you?

**Comment by konsumer** [comment]
Seems to be working with wrangler@alpha. Thanks!
